src/target.rs

```diff
@@ -145,6 +146,7 @@ impl Target {
             os,
             arch,
             env: platform.target_env,
+            triple: platform.target_triple,
```

Comment:
I hope that this is really the same target as cargo uses and we don't start cross-compiling where we wouldn't need to :crossed_fingers: